### ISSUE TITLE
Specific versions of dependencies

### ISSUE THREAD

**Issue by lafrenierejm** [guyzmo/git-repo#133]
Would it be feasible to unlock the version requirements on uritemplate and github3?  Do the latest version of those packages (3.0.0 and 1.0.0a4, respectively, as of this comment) actually break anything?

**Comment by lafrenierejm** [comment]
Reading through some of the open PRs I see that @guyzmo is aware of the problem.  Would someone be willing to provide a rundown of the different issues/features blocking versions on each requirement?

**Comment by guyzmo** [comment]
well, long story short, uritemplate.py has been renamed in uritemplate, using a hack that breaks raw setuptools. Fixing the version solves this temporarily.

About github3.py, AFAICT, latest version is still <1.0.0, and 1.0.0 is introducing breaking changes that will need rewrite of parts of the library. I'm waiting on @sigmavirus24 to give the green light for starting switching the API (shouldn't be long).

Basically this situation is temporary and should resolve soon ☺ But it's stupid that it breaks availability on for non-pypi distribution (like Arch).

**Comment by lafrenierejm** [comment]
> About github3.py, AFAICT, latest version is still <1.0.0, and 1.0.0 is introducing breaking changes that will need rewrite of parts of the library.

You are indeed correct; the current 1.0.0 releases, despite being tagged in Git and available on PyPi, are alpha.  Thank you for correcting my understanding there.
